.github/workflows/release.yml

```diff
@@ -0,0 +1,133 @@
+name: Release
+
+on:
+  # pull_request:
+    # branches: [ master ]
```

Comment:
What are your intentions of having this run on PRs against master? Or was this just for debugging?

Reply:
No intention it’ll be ripped out that’s what I was testing it before I switch to pushing actual tags.